Align to OTP 24

KornelH opened the pull request:
Fix below compile-time warning. The recommended erlang:crc32/1 was introduced in Erlang/OTP R12.

```
egd_png.erl:102:11: zlib:crc32/2 is deprecated and will be removed in OTP 27; use erlang:crc32/1 instead
```

KornelH committed:
Align to OTP 24

Fix below compile-time warning. The recommended erlang:crc32/1 was
introduced in Erlang/OTP R12.

egd_png.erl:102:11: zlib:crc32/2 is deprecated and will be removed in
                    OTP 27; use erlang:crc32/1 instead

newmancodes commented:
Is it possible to get this change merged? Working through some learning materials and this is a blocker for a few people.

mcampbell commented:
> Is it possible to get this change merged? Working through some learning materials and this is a blocker for a few people.

I've run into what appears to be this issue with an Udemy Elixir course using egd, on the latest OTP.

anthonyhoegberg commented:
Im getting this issue aswell can this be merged?

savs2020 commented:
Reinstall Elixir v1.17 with Erlang/OTP 26.

KornelH committed:
Version 0.10.1

KornelH committed:
Update documentation for ex_doc

Documentation provider is changed to ex_doc [1] that required to:
- add a missing spec line to fix an ex_doc error, and
- convert erl_docgen [2] style XML documents to Markdown.

[1] https://hexdocs.pm/rebar3_ex_doc/
[2] https://www.erlang.org/docs/26/man/erl_docgen_app.html#

anthonyhoegberg commented:
Looks like this repository might be abandoned? any ideas of how to preceded with such a case?

jgaik commented:
I am currently using the forked version from @KornelH who applied the change there. Maybe there is someone from erlang community who can proceed with this but I am new here 😓

KornelH commented:
EGD is an old an unmaintained library, but still in use by some. So, I can publish the updated version on [hex.pm](https://hex.pm/) to be able to use it via [rebar3](https://rebar3.org/) easily. As far as I understand, the license allows it.

KornelH commented:
Now, it is published at [hex.pm](https://hex.pm) as package [egd24](https://hex.pm/packages/egd24) and can be included into any [rebar3](https://rebar3.org/) project by adding below dependencies to `rebar.config`:

```erlang
{deps, [
    {egd, {pkg, egd24}}
]}.
```
